.github/workflows/release-secure-partitions.yml

```diff
@@ -17,6 +17,7 @@ on:
     paths:

```

Comment:
Can you please mention in the PR what specific toml files this is targeting, for clarity's sake?